src/scripts/notebooks/clinvar-udn-submission-analysis/data/output/clinvar_no_mondo.tsv

```diff
@@ -0,0 +1,423 @@
+pref_name
+ORC3-related disorder
+ADCY2-related disorder
+ZNF331 deletion
+NOTCH1-related disorder
+KDR-related disorder
+KIF5B-related osteogenesis imperfecta syndrome
+PAPSS1-related disorder
+MYCBP2-related disorder
+TMEM161B-related lissencephaly
+DHX9-related disorder
+DAGLA-related disorder
+SPTSSA-related disorder|Spastic paraplegia 90A, autosomal dominant
+ZNF865-related disorder
+SREBF2-related disorder
+Neurodevelopmental delay|Seizure
+Spastic ataxia 5|not provided
+RRAGD-related disorder|Hypomagnesemia 7, renal, with or without dilated cardiomyopathy
+not provided|PLA2G6-associated neurodegeneration
+DPH5-related diphthamide-deficiency syndrome|Neurodevelopmental disorder with short stature, prominent forehead, and feeding difficulties
+IQGAP1-associated immune condition
+PPP5C-related disorder|not provided
+CHARGE association
+MIR145-related multisystemic smooth muscle dysfunction
+ATP5F1A-related disorder
+NAV2-related neurodevelopmental condition
+Basal ganglia calcification, idiopathic, 5|not provided
+PHACTR1-related neurodevelopmental condition
+ATAD3 gene cluster related condition
+Spastic paraplegia 84, autosomal recessive|PI4KA-related disorder
+not provided|Primary ciliary dyskinesia
```

Comment:
What does "not provided" mean?

Reply:
not sure, can be addressed if/when this work moves forward